_data/SamReeve96/SamReeve96.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Samuel Reeve
+institution: University of Portsmouth
+profile_pic: SamuelReeve.JPG
+quote: The Best Thing To Do Is Stop Trying To Figure Out Where You’re Going And Just Enjoy Where You’re At
```

Comment:
Change quote ’ from non-ascii to ascii ' character.

Reply:
Hi, thanks for pointing this one out. It should be resolved as of commit - https://github.com/education/graduation/pull/4125/commits/7e86d00c7f9be9485906d06a37461a804c253bb4